run_flask.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/python3
```

Comment:
This should be `#! /usr/bin/env python3`
Trying to keep it consistent across all the executables